Emby.Server.Implementations/SystemManager.cs

```diff
@@ -72,16 +74,34 @@ public SystemInfo GetSystemInfo(HttpRequest request)
         };
     }
 
-    /// <inheritdoc />
+    /// <summary>
+    /// Gets public system info of the server.
+    /// LocalAddress is the server URL that can also be used by applications not on the server, for instance
+    /// a cast receiver device. In case the request has a localhost url, LocalAddress will be set to a
+    /// server Private network address (IPv4) or Unique Local Address (ULA, IPv6) which is valid for all
+    /// devices on the LAN.
+    /// </summary>
+    /// <param name="request">Where this request originated.</param>
+    /// <returns>PublicSystemInfo.</returns>
     public PublicSystemInfo GetPublicSystemInfo(HttpRequest request)
     {
+        string ip4LoopBk = Network.IPv4RFC5735Loopback.Prefix.ToString();
+        string ip6LoopBk = "[" + Network.IPv6RFC4291Loopback.Prefix.ToString() + "]";
+        string host = request.Host.Host;
```

Comment:
Can't really trust this as it's a header. If possible, it's better to use `request.HttpContext.Connection.RemoteIpAddress`